tests/ext/used_dispatch_shouldn_t_be_freed.phpt

```diff
@@ -0,0 +1,22 @@
+--TEST--
+xxx
```

Comment:
Looks like we're missing a little bit more of a description here. 😆

Reply:
🤦‍♂️ 
Thanks for spotting that :D